### ISSUE TITLE
Rise of Shadows patch 14.0.0.29933 released

### ISSUE THREAD

**Issue by sebastientromp** [schmich/hearthstone-card-images#19]
Hey, you're probably aware of it but I just wanted to let you know that the latest patch (14.0.0.29933) for Rise of Shadows was released yesterday.

Cheers!

**Comment by schmich** [comment]
Yes, definitely! I will get these published soon.

**Comment by amarriner** [comment]
Do extract these out of the client automatically or do you assemble them manually?

**Comment by flowdee** [comment]
Any update on this? Eagerly waiting for the new images :)

**Comment by schmich** [comment]
I'll copy-paste my response from #22:

My apologies on this. I've moved this project over to use the [official Hearthstone API](https://develop.battle.net/documentation/api-reference/hearthstone-game-data-api) for images, so keeping cards up-to-date will be _significantly_ easier moving forward. This project is now pretty redundant with what Blizzard themselves offer, but I will keep it going.

I've updated the card images up through the latest Saviors of Uldum release.

Also, all of the card image fetching/updating/maintenance is now part of this repo under the [update](https://github.com/schmich/hearthstone-card-images/tree/master/update) folder.
